backend/spec/requests/comments_spec.rb

```diff
@@ -33,4 +33,60 @@
       end
     end
   end
+
+  describe 'PUT /memos/:memo_id/comments/:id' do
+    context 'コンテンツが有効な場合' do
+      let(:memo) { create(:memo) }
+      let(:comment) { create(:comment, memo: memo) }
+      let(:valid_comment_params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
```

Comment:
Nits

名前が少し冗長かなと思いました！
paramsでいいかなと！

```suggestion
      let(:params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
```